src/environments/environment.ts

```diff
@@ -8,5 +8,5 @@ import { config } from './config';
 export const environment = {
   production: false,
   config: config,
-  endpoint: 'ws://localhost:4000',
+  endpoint: 'ws://localhost:6032',
```

Comment:
Please remove this line, as it will break the docker build

Reply:
You may wanna change it yourself as I won’t be able within the next 24h.

Reply:
Done